Packages/cc.starlessnight.unity-jsb/Source/Binding/DynamicMethod.cs

```diff
@@ -169,7 +169,11 @@ public override JSValue Invoke(JSContext ctx, JSValue this_obj, int argc, JSValu
                     }
                     else
                     {
-                        if (!Values.js_get_var(ctx, argv[argvIndex], pType, out args[paramIndex]))
+                        if (argvIndex >= argv.Length)
+                        {
+                            args[paramIndex] = null;
+                        }
+                        else if (!Values.js_get_var(ctx, argv[argvIndex], pType, out args[paramIndex]))

```

Comment:
This prevents an IndexOutOfRangeException when calling a C# method with optional arguments from Javascript.